code/web/src/modules/user/test/__snapshots__/profile.test.js.snap

```diff
@@ -0,0 +1,89 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Profile should display profile text and buttons 1`] = `
+<div>
```

Comment:
Is this code like "dummy" code that is setup for a jest test? File name says snapshots, so it reminds me of a dummy file to test against